great-idea-website/index.html

```diff
@@ -14,50 +14,63 @@
   <![endif]-->
 </head>
 
-  <body>
-
-  Services
-  Product
-  Vision
-  Features
-  About
-  Contact
-
-  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-  Innovation
-  On
-  Demand
-
-  Get Started
-
-  <img src="img/header-img.png" alt="Image of a code snippet.">
-
-  Features
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-
-  About
-  Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
-
-  <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+<body>
+  <header>
+    <nav>
+      <a href="#">Services</a>
+      <a href="#">Product</a>
+      <a href="#">Vision</a>
+      <a href="#">Features</a>
+      <a href="#">About</a>
+      <a href="#">Contact</a>
+    </nav>
+    <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+  </header>
+  <section class="splashPage">
+    <div class="left-splash-wrapper">
+      <h1>Innovation
+      <br>On
+      <br>Demand</h1>
+    
+    
+
+      <button>Get Started</button>
+    </div>
```

Comment:
Good use of semantic tag names here.
Your code is indented properly making it easier to read.
Are you using a formatter like beautify or prettier? they are available extensions tab.
You can set it up to format on save or on a hotkey combination like shift+alt+f